index.html

```diff
@@ -0,0 +1,186 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Eastern Designs</title>
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+  <link href="style.css" rel="stylesheet"/>
+  <link href="navbar.css" rel="stylesheet"/>
+  <script src="https://kit.fontawesome.com/a012cea4ea.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+<header class="header">
+  <nav class="navbar">
+      <a href="#" class="nav-logo">WebDev.</a>
+      <ul class="nav-menu">
+          <li class="nav-item">
+              <a href="#" class="nav-link">Services</a>
+          </li>
+          <li class="nav-item">
+              <a href="#" class="nav-link">Blog</a>
+          </li>
+          <li class="nav-item">
+              <a href="#" class="nav-link">About</a>
+          </li>
+          <li class="nav-item">
+              <a href="#" class="nav-link">Contact</a>
+          </li>
+      </ul>
+      <div class="hamburger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+      </div>
+  </nav>
+</header>
+<div>
+  <ul class="categories">
+    <li>Shirts</li>
+    <li>Sneakers</li>
+    <li>Shorts</li>
+    <li>Caps</li>
+  </ul>
+</div>
+<div class="best-seller-section-container">
+  <div class="best-seller-section">
+    <h2>Best Sellers</h2>
+    <span class="see-wrapper">
+      <p>See all </p>
+      <span class="material-icons">chevron_right</span>
+    </span>
+  </div> 
+</div>
+<div class="card-holder">
+    <div class="card">
+      <img src="img/one-blood.JPG" alt="One Blood T-Shirt & Sneaker" class="project-img">
+        <ul>
+          <div class="rating-container">
+            <span class="material-icons rate checked">star_outline</span>
+            <span class="material-icons rate checked">star_outline</span>
+            <span class="material-icons rate checked">star_outline</span>
+            <span class="material-icons rate">star_outline</span>
+            <span class="material-icons rate">star_outline</span>
+          </div>
+          <h3 class="card-header">Clothing</h3>
+          <li>Sneakers & T Shirt</li>
+          <li class="price">K250</li>
+        </ul>
+    </div>

```

Comment:
- [ ] Kindly change the tag from `div` to a more semantic tag `<card>`, this is one of the best practices for code readability. Please take a look at this article to learn more about the benefits of using [semantic tags](https://developer.mozilla.org/en-US/docs/Learn/Accessibility/HTML)